### PR TITLE
Change Edit path for Collections

### PR DESCRIPTION
**Pull request by njaffer**
Fixes this error:
screen shot 2016-06-17 at 2 25 15 pm

Changes proposed in this pull request:
* update edit path for collection
* 
* 

@projecthydra/sufia-code-reviewers

### CODE REVIEW

**Comment by awead** [comment]
Would you be willing to write a short test that verifies this? You should be able to crib from https://github.com/projecthydra/curation_concerns/blob/master/spec/views/catalog/_action_menu_partials/_default.html.erb_spec.rb

**Comment by awead** [comment]
I'm also willing to forgo the test, too 😄

**Comment by njaffer** [comment]
added the test..

**Comment by jcoyne** [comment]
I think the problem here is that the `document` does not have an id. So it's trying `edit_collections_path` (note the plural) and not `edit_collection_path` as it should.

File: spec/views/catalog/_action_menu_partials/_collection.html.erb_spec.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe 'catalog/_action_menu_partials/_collection.erb' do
+  let(:document) { double }
```
Comment (jcoyne) [inline spec/views/catalog/_action_menu_partials/_collection.html.erb_spec.rb]: It can't possibly draw the correct route without an identifier.

**Pull request closed without merging**